Components/Alert/Alert.html

```diff
@@ -0,0 +1,140 @@
+  </head>
+  <body>
+    <h1>Alerts</h1>
+    <h2>Simple Alerts</h2>
+    <div class="alerts">
+      <div class="error-alert-background alert">
+        <svg
+          class="alert-image"
+          focusable="false"
+          viewBox="0 0 24 24"
+          aria-hidden="true"
+        >
+          <path
+            d="M11 15h2v2h-2zm0-8h2v6h-2zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
+          ></path>
+        </svg>
+
+        <span class="alert-text">This is an error alert</span>
+      </div>
+      <div class="warning-alert-background alert">
+        <svg
+          class="alert-image"
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke="currentColor"
+        >
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            stroke-width="2"
+            d="M12 9v2m0 4h.01m-6.938 4h13.856c1.54 0 2.502-1.667 1.732-3L13.732 4c-.77-1.333-2.694-1.333-3.464 0L3.34 16c-.77 1.333.192 3 1.732 3z"
+          />
+        </svg>
+
+        <span class="alert-text">This is a warning alert</span>
+      </div>
+      <div class="info-alert-background alert">
+        <svg
+          class="alert-image"
+          focusable="false"
+          viewBox="0 0 24 24"
+          aria-hidden="true"
+        >
+          <path
+            d="M11,9H13V7H11M12,20C7.59,20 4,16.41 4,12C4,7.59 7.59,4 12,4C16.41,4 20,7.59 20, 12C20,16.41 16.41,20 12,20M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10, 10 0 0,0 12,2M11,17H13V11H11V17Z"
+          ></path>
+        </svg>
+
+        <span class="alert-text">This is an info alert</span>
+      </div>
+      <div class="success-alert-background alert">
+        <svg
+          class="alert-image"
+          focusable="false"
+          viewBox="0 0 24 24"
+          aria-hidden="true"
+        >
+          <path
+            d="M20,12A8,8 0 0,1 12,20A8,8 0 0,1 4,12A8,8 0 0,1 12,4C12.76,4 13.5,4.11 14.2, 4.31L15.77,2.74C14.61,2.26 13.34,2 12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0, 0 22,12M7.91,10.08L6.5,11.5L11,16L21,6L19.59,4.58L11,13.17L7.91,10.08Z"
+          ></path>
+        </svg>
+
+        <span class="alert-text">This is a success alert</span>
+      </div>
+    </div>
+    <h2>Descriptive Alerts</h2>
+    <div class="alerts">
+      <div class="error-alert-background alert">
+        <svg
+          class="alert-image"
+          focusable="false"
+          viewBox="0 0 24 24"
+          aria-hidden="true"
+        >
+          <path
+            d="M11 15h2v2h-2zm0-8h2v6h-2zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
+          ></path>
+        </svg>
+        <span class="alert-header">Error</span>
+        <div class="desc-alert-text">This is an error alert</div>
```

Comment:
desc gives the context of the class name, text can be removed for consistency.
```suggestion
        <div class="alert-desc">This is an error alert</div>
```